### ISSUE TITLE
Input-shape

### ISSUE THREAD

**Issue by doaaobeidat** [ZFTurbo/classification_models_3D#21]
Hi, If I want to use CT scans , so the width is 128, length 128 and the depth for example 96, how can  I use Resnet

**Comment by ZFTurbo** [comment]
Like this:

```
from classification_models_3D.tfkeras import Classifiers

ResNet18, preprocess_input = Classifiers.get('resnet18')
model = ResNet18(input_shape=(128, 128, 96, 1), weights='imagenet')
```

**Comment by doaaobeidat** [comment]
When I made what you have mentioned it gives me error about unequal axis, how can I can use these models if I want to use input_shape=(128, 128, 96, 1) and input_shape=(128, 128, 96, 3)

**Comment by ZFTurbo** [comment]
I need more details about your error.

**Comment by doaaobeidat** [comment]
From what I gather from your explanation, it seems that I can utilize your pretrained models, which have already undergone the conversion of image weights from 2D to 3D. Therefore, my implementation would look something like this:
from keras_applications import Classifiers
from tensorflow.keras.layers import GlobalAveragePooling3D, Dense, Dropout
from tensorflow.keras.models import Model

# Retrieve the DenseNet121 model and its associated preprocessing function
densenet121, preprocess_input = Classifiers.get('densenet121')

# Create my model architecture based on the pretrained DenseNet121
model = densenet121(input_shape=(96, 96, 96, 1), weights=None, include_top=False)
x = GlobalAveragePooling3D()(model.output)
x = Dense(units=256, activation="relu")(x)
x = Dropout(0.7)(x)
outputs = Dense(units=1, activation="sigmoid")(x)

# Assemble the final model
model = Model(model.input, outputs=outputs, name="densenet121")
<img width="960" alt="code" src="https://github.com/ZFTurbo/classification_models_3D/assets/73997623/7361040c-2cbd-4ac7-9314-96b5510d892c">
<img width="960" alt="error" src="https://github.com/ZFTurbo/classification_models_3D/assets/73997623/d8062e38-883d-49f7-9508-1f53d6bcc1b2">
<img width="960" alt="input" src="https://github.com/ZFTurbo/classification_models_3D/assets/73997623/b5054c57-381b-4836-ac0d-826165b3f76a">

**Comment by ZFTurbo** [comment]
To use imagenet weights you need to use shape which has 3 channels: "(96, 96, 96, 3)", it's actually not to much complicate model - only 1st layer changed. You can duplicate input 3 times with `x = np.stack([x, x, x], axis=-1)`
